account_banking_pain_base/models/account_payment_order.py

```diff
@@ -706,3 +709,20 @@ def generate_creditor_scheme_identification(
         csi_scheme_name_proprietary = etree.SubElement(csi_scheme_name, "Prtry")
         csi_scheme_name_proprietary.text = scheme_name_proprietary
         return True
+
+    @api.depends("payment_mode_id")
+    def _compute_batch_booking(self):
+        self.batch_booking = False
+        if self.payment_mode_id:
+            self.batch_booking = self.payment_mode_id.default_batch_booking
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        payment_mode_model = self.env["account.payment.mode"]
+        for vals in vals_list:
+            if "batch_booking" not in vals:
```

Comment:
Now with the computed writable, this shouldn't be needed.

Reply:
Was not sure  in fact and probably a bit lazy. Sorry.